rest-api/src/main/resources/definitions/paged_resources/forward_cursor_paged_resource_list.yml

```diff
@@ -0,0 +1,21 @@
+type: object
+discriminator: items
+description: |
+    One page of records from a source that only allows you to fetch the next page of 
+    resources (no random access via an index to anywhere in the set of records). The 
+    key to retrieve the next page of records is provided by the `offsetKey` value.
+properties:
```

Comment:
Is it worth adding the interim properties (like pageSize, assignmentFilter, idFilter, etc)? Do we know if we'll break anything on the Android side of things if we just pull these properties out straight away?